src/AppEntry.js

```diff
@@ -10,17 +10,16 @@ import { Provider } from 'react-redux';
 import { BrowserRouter as Router } from 'react-router-dom';
 import { cache } from './store/cache';
 import { getBaseName } from '@redhat-cloud-services/frontend-components-utilities/helpers/helpers';
-import { getRegistry } from '@redhat-cloud-services/frontend-components-utilities/Registry';
-import { init } from './store';
+import { init, RegistryContext } from './store';
 import logger from 'redux-logger';
 import messages from '../locales/data.json';
 import { setContext } from '@apollo/client/link/context';
 
 const AppEntry = ({ useLogger, connectToDevTools }) => {
     const tags = useRef();
+    const registry = useLogger ? init(logger) : init(logger);
```

Comment:
No need to have logger in production
```suggestion
    const registry = useLogger ? init(logger) : init();
```